### ISSUE TITLE
About Model Ensemble

### ISSUE THREAD

**Issue by chaolongy** [dvlab-research/Stratified-Transformer#55]
Hi, can you be specific about the detailed process of the model ensemble in the supplementary material?https://openaccess.thecvf.com/content/CVPR2022/supplemental/Lai_Stratified_Transformer_for_CVPR_2022_supplemental.pdf.

**Comment by X-Lai** [comment]
For model ensemble, we averaged the results of four models trained with different hyper-parameters, e.g., batch_size, weight_decay, learning rate.
